lib/src/database/db.dart

```diff
@@ -209,7 +210,7 @@ class Db {
       }
 
       if (connection == null) {
-        connection = _masterConnection;
+        connection = _masterConnectionVerified;
```

Comment:
Just bump the version of the package as a minor and I'll merge. I believe it's gonna be hard to write a test for this after all.